CHANGELOG.md

```diff
@@ -11,10 +11,11 @@
 
 **Bug Fixes**
 
+- Fix when removing `collectionName` from user role [\#1471](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1471) [Breaking Change] ([marcosuma](https://github.com/marcosuma)).
+  - Note: As a result, `mongodbatlas_database_user` is updated to not require `roles.collection_name` attribute and not support empty `collection_name`. Users should remove any usage of `roles.collection_name = ""` in configuration for this resource for upgrading to this version. For more details see:  https://registry.terraform.io/providers/mongodb/mongodbatlas/latest/docs/guides/1.12.0-upgrade-guide 
```

Comment:
tweak to remove passive voice + imperative form to align with company style + end last sentence with a period
```suggestion
  - Note: As a result, `mongodbatlas_database_user` no longer requires `roles.collection_name` attribute and doesn't support an empty `collection_name`. You should remove any usage of `roles.collection_name = ""` in configurations for this resource when you upgrade to this version. For more details see:  https://registry.terraform.io/providers/mongodb/mongodbatlas/latest/docs/guides/1.12.0-upgrade-guide. 
```